packages/catalog-model/examples/components/artist-lookup-component.yaml

```diff
@@ -28,6 +28,8 @@ metadata:
     - url: https://example.com/alert
       title: Alerts
       icon: alert
+  annotations:
+    backstage.io/linguist: 'https://github.com/backstage/backstage/tree/master/plugins/playlist' 
```

Comment:
Yes, that's something that I've considered on and off. Guess my worry is that it will try to generate all this data for a large catalog and maybe that's not what people might expect? Our Catalog is pretty small in the grand scheme of things and though this works really well for us I'm not sure how this would react with a really big catalog.

I'm leaning towards this being a config option that defaults to using the `backstage.io/linguist` annotation.

Reply:
With my refactor of the processing I brought this in as an option